internal/command/context_test.go

```diff
@@ -0,0 +1,66 @@
+package command
+
+import (
+	"reflect"
+	"testing"
+)
+
+func Test_makeDefaultContext(t *testing.T) {
+	type args struct {
+		flags *CommonFlags
+	}
+	tests := []struct {
+		name        string
+		args        args
+		wantEnv     []string
+		wantLdFlags []string
```

Comment:
```suggestion
		want Context
		wantErr bool
```
what about testing directly the function output ?

Reply:
Anyway the fix is covered and I'll have to add other test cases to this due to another bug so I'm going to merge as is ;)
Thanks!